actix-server/src/signals.rs

```diff
@@ -1,8 +1,10 @@
 use std::future::Future;
-use std::io;
 use std::pin::Pin;
 use std::task::{Context, Poll};
 
+#[cfg(not(unix))]
+use std::io;
```

Comment:
inline this since it's only used in one place under the cfg attr

Reply:
this has come back

Reply:
Sorry :parachute:  Now it's fixed